Source/Scripts/cheat_core_items.lua

```diff
@@ -322,9 +322,25 @@ cheat:createCommand("cheat_own_all_stolen_items", "cheat:cheat_own_all_stolen_it
   "Makes you the owner of all stolen items in your inventory.\n$8This removes the stolen flag from the item.",
   "Take ownership of stolen items", "cheat_own_all_stolen_items")
 function cheat:cheat_own_all_stolen_items()
-  cheat:recreateitems("ownstolen")
```

Comment:
What was broken with this command? Your code looks almost the same except you bulk remove everything in the inventory.

Reply:
It was crashing the all time all the time after used the command.

On my tests, i noticed that the line:
local itemOwner = tonumber(string.gsub(tostring(ItemManager.GetItemOwner(item.id)), "userdata: ", ""), 10)

Isn't getting the right value. I think. I don't understand enough the code.
I found that item.id is giving the value:
['id'] = 'userdata: 0200000000011518',

I don't understand the conversion tonumber with base 10? am i right? , it's giving a nil value, i dont know the reason.
So i set all the items into one list/array/object with health status and amount, removed all items, and created new ones, but now the owner is you/Henry so they aren't stolen.
This is working and isn't crashing the game anymore ;)

I prefer use the command to just remove the stolen status of the item, but i have no idea how get the owner/stolen status and don't understand the id of the item to use.

I find the Lua class:
```
ItemManager.SetItemOwner(itemId, ownerId, markAsStolen)
Parameters

    itemId: A string representing the UUID of an item
    ownerId: A string representing the UUID of the item's new owner
    markAsStolen A boolean determining whether the item is stolen

Return value

nil
```
But i didn't figure how use that. How i get the itemID (using the for on each item, because item userdata userdata gives me only those options:
```
{
	['health'] = 1,
	['amount'] = 1,
	['id'] = 'userdata: 0200000000011518',
	['class'] = '4dd7d65f-26ee-8189-d398-10ed812b6e98',
	['entity'] = 0
}
```

If you know, could you give me one working e.g. on how use the class: 
- How check if iten has stolen and if possible, dirty status.
- How use correct the class: ItemManager.SetItemOwner()
- How check if the item is eqquiped or not
- And how use the class: player.actor:EquipInventoryItem(id) too.
I didn't find any guide how use those classes.

But still I can fixed removing all items and recreating again ;)
The only trouble is that this way Henry unequip all items. But not a really big deal.
It isn't crashing the game (the game isn't closing anymore) so we can use the command with the fix with no problem.
tested with latest steam v1.9.6 version of the game.

Reply:
It isn't a string 'userdata: 0200000000011518' that's just the string representation for userdata, which are custom C pointers, like tables if the metatable is defined for it, but could generally be used for anything.

Reply:
But yes, this function does crash the game in the current shipped state